Dockerfile.dapper

```diff
@@ -5,6 +5,14 @@ ENV DAPPER_ENV="REPO TAG QUAY_USERNAME QUAY_PASSWORD GITHUB_REF CLUSTERS_ARGS DE
     DAPPER_SOURCE=/go/src/github.com/submariner-io/submariner DAPPER_DOCKER_SOCKET=true
 ENV DAPPER_OUTPUT=${DAPPER_SOURCE}/output
 
+ENV PROTOC_VERSION=3.15.5
+
+# alternative: dnf install -y --nodocs --setopt=install_weak_deps=False golang-google-protobuf protobuf-compiler && dnf -y clean all
```

Comment:
Just a side note: downloading and installing manually takes less than 20s, going through dnf outside Shipyard takes over a minute (because the repo information needs to be downloaded).

Reply:
In my case it was more than 3 minutes, may be a bad network day..